telosnetwork/open-block-explorer
# #630 | Resources loading error

# Fixes #630 

## Description
This PR adds a loading state for Resources and Staking buttons on the account view

## Test scenarios
- go to https://deploy-preview-632--obe-testnet.netlify.app/
- log in
- go to your account page
    - while the page is loading, all the buttons should be disabled and say "Loading"

## Checklist:
